src/InputManager.js

```diff
@@ -0,0 +1,39 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+import Validator from './utils/Validator.js';
+import converStringToNumber from './utils/convertStringToNumber.js';
+import splitStringByComma from './utils/splitStringByComma.js';
+
+class InputManager {
+  static async getCarNames() {
+    const carNamesInput = await this.readCarNamesInput();
+    const carNames = splitStringByComma(carNamesInput);
+    this.validateCarNames(carNames);
+    return carNames;
+  }
+
+  static async readCarNamesInput() {
+    return MissionUtils.Console.readLineAsync(
+      '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n',
+    );
+  }
+
+  static validateCarNames(carNames) {
+    Validator.validateCarNames(carNames);
+  }
+
+  static async getAttempts() {
+    const attemptsInput = await this.readAttemptsInput();
+    this.validateAttempts(attemptsInput);
+    return converStringToNumber(attemptsInput);
```

Comment:
해당 코드도 위 리뷰랑 마찬가지입니다!

굳이 converStringToNumber이라는 함수를 따로 만들어야 되나? 라는 생각이 들었어요! 해당 함수가 하는 일은 결과적으로 Number()로 감싼 값을 리턴하는데 그러면 아래 처럼 작성하는 것이 추가적인 함수를 만들지 않아도 되고 가독성 측면에서도 더 좋다고 생각이 들었습니다!
```suggestion
    return Number(attemptsInput);
```

Reply:
나중에 재사용 할 수 있을 것 같아서 따로 분리한건데, 분리하지 않고 한 번에 바로 하는게 의미가 더 잘 전달되고 직관적인 것 같네요! 좋은 의견 감사합니다 :)